test/unit/cog_api/http/bundles_test.exs

```diff
@@ -65,6 +66,28 @@ defmodule CogApi.HTTP.BundlesTest do
       end
     end
 
+    context "when given a bundle name to be updated" do
+      it "returns the updated bundle" do
+        cassette "bundles_update" do
+          endpoint = valid_endpoint
+
+          mist_bundle = Client.bundle_index(endpoint)
```

Comment:
I could if I had the bundle_id, but I don't at this point of the test. This is finding a bundle that I can update. I then update it via it's name instead of it's id.